web/src/app/escalation-policies/PolicyStepsCard.tsx

```diff
@@ -122,17 +116,10 @@ export default function PolicyStepsCard(
       )}
       {createStep && (
         <React.Fragment>
-          {hasDestTypesFlag ? (
-            <PolicyStepCreateDialogDest
-              escalationPolicyID={props.escalationPolicyID}
-              onClose={resetCreateStep}
-            />
-          ) : (
-            <PolicyStepCreateDialog
-              escalationPolicyID={props.escalationPolicyID}
-              onClose={resetCreateStep}
-            />
-          )}
+          <PolicyStepCreateDialogDest
```

Comment:
I think I would be fine having these take the old names now that we're removing it, for sake of brevity 🙂 Thoughts?

```suggestion
          <PolicyStepCreateDialog
```

Reply:
How would you feel about doing a rename after all at once? if we do them at the same time, git will think we're making a bunch of changes instead of just deleting the old versions. in hindsight we could have renamed the old to `*Legacy` or something so it's clear they would be deleted in the future

Reply:
Yeah that works 👍